README.md

```diff
@@ -26,28 +27,27 @@ See our [getting started with flintlock][quickstart] guide.
 
 ## Contributing
 
-Contributions are welcome. Please read the [CONTRIBUTING.md][contrib] and our [Code Of Conduct][coc].
+Contributions are welcome. Please read the [CONTRIBUTING.md][contrib] and our [Code Of Conduct][coc]. We have the [#liquid-metal](https://weave-community.slack.com/archives/C02KARWGR7S) slack channel for discussions when contributors and users.
```

Comment:
> slack channel for discussions when contributors and users
The `when` feels odd here, or the sentence is not complete?

Reply:
You're right, it is very weird.

Reply:
Changed